CMakeLists.txt

```diff
@@ -176,6 +176,7 @@ if(UNIX AND NOT APPLE AND NOT ANDROID AND NOT HAIKU)
             add_definitions(-DHAVE_LIBSECRET=1)
         endif()
         INCLUDE_DIRECTORIES(${LIBSECRET_INCLUDE_DIRS})
+	LINK_DIRECTORIES(${LIBSECRET_LIBRARY_DIRS})
```

Comment:
whitespace seems off (tab vs. spaces?)
Can we use target_link_libraries here without using link_directories directly? see note in https://cmake.org/cmake/help/latest/command/link_directories.html

Reply:
We can and I would use it, but it would require to reorganize the file to link the libraries after the target was defined.
Then we could handle the defines and include dirs the same way.

For whitespace, github editor....

Reply:
White space fixed